src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/dto/CcdBundleDTO.java

```diff
@@ -13,13 +13,15 @@
     private String title;
     private String description;
     private String eligibleForStitching;
+    private String eligibleForCloning;
     private CcdDocument stitchedDocument;
     private List<CcdValue<CcdBundleDocumentDTO>> documents = new LinkedList<>();
     @JsonIgnore
     private List<CcdValue<CcdBundleFolderDTO>> folders = new LinkedList<>();
     private String fileName;
     private CcdBoolean hasTableOfContents;
     private CcdBoolean hasCoversheets;
+    private String stitchStatus;
```

Comment:
Where did this come from?

Reply:
I realised that the CCD definition file has stitchStatus, but the CcdBundleDTO doesn't, so I added it. Otherwise, CCD has information that never makes it over to something like the Stitching API